se702-cs706/virtual-pet-game-se702cs705
# Feature character connect to firebase

Added a firebase script for logging information to firebase.

TEST:
- Run a method in awake to update or add a logging object to firebase
- Check firebase that it pushed properly
- Delete the code and document on firebase to preserve the initial state
